scripts/setup-kind-multicluster.sh

```diff
@@ -153,7 +153,7 @@ function deploy_cass_operator() {
 
   for ((i=0; i<$num_clusters; i++))
   do
-    kustomize build config/cass-operator | kubectl --context kind-k8ssandra-$i apply -f -
+    kustomize build config/cass-operator/cluster-scoped | kubectl --context kind-k8ssandra-$i apply -f -
```

Comment:
The `deploy_cass_operator` function isn't used since the script doesn't deploy cass-operator. It did in an earlier version though. Can you remove the function?

Reply:
Confirming my understanding here: `deploy_cass_operator` is a function which is no longer called anywhere in the codebase and needs to be removed?

Reply:
I was just updating it so that we had consistency, but if it is no longer used it makes sense to get rid of it.

Reply:
John and I have spoken offline and this function should be deleted.

Reply:
I've implemented this change. Was there anything else required to progress this @jsanda?